osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -430,13 +433,11 @@ private void clearPlate(DroppedObjectAnimation animation)
 
         private void removeFromPlate(CaughtObject caughtObject, DroppedObjectAnimation animation)
         {
-            var droppedObject = getDroppedObject(caughtObject);
-
             caughtObjectContainer.Remove(caughtObject, false);
 
-            droppedObjectTarget.Add(droppedObject);
+            droppedObjectTarget.Add(getDroppedObject(caughtObject));
 
-            applyDropAnimation(droppedObject, animation);
+            applyDropAnimation(caughtObject, animation);
```

Comment:
Readability wise this doesn't make muich sense to me. Why is the drop animation applied to the original object here?

Reply:
Oh I completely missed this, my bad.